### PR TITLE
Chirpus-lab-first-commit

### PR DESCRIPTION
**Pull request by Jpropst**
Chirpus lab first commit

### CODE REVIEW

**Comment by Jsmoove13** [comment]
Chirpus lab first commit

File: chirpus/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <button class="flat-btn">Hire Us</button>
+    </div>
+    <div class="wrapper">
+        <h1>chirpus inc.</h1>
+        <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ex vel maxime nesciunt iusto est optio labore alias quo quae expedita modi animi ea, incidunt velit nemo laborum cumque beatae ab iste tenetur cum? Laborum corporis vel at tempore blanditiis quisquam totam ratione aspernatur excepturi impedit dolorum officia, omnis hic. Adipisci.</p>
+        <button class="flat-btn">Call to Action</button>
+    </div>
+</body>
+<footer>
+    <div class="footer-left-link">chirpus.inc</div>
```
Comment (KatSchultz) [inline chirpus/index.html:26]: Small typo, should be "chirpus inc."

**Comment by KatSchultz** [comment]
Really good job matching the display. 

Be sure to go over the build specifications in the lab before submitting. They give specific colors to use. When using black, your CSS has both "#000" and "black". The specs requested "#FFF" for button text, but you used "white". 

You hit all the other specs, so great work overall.

File: chirpus/index.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400&family=Roboto&display=swap');
```
Comment (KatSchultz) [inline chirpus/index.css:1]: You shouldn't have to import the font like this in CSS if you have already imported it in the head of HTML.

**Review by KatSchultz — APPROVED** [review]

**Pull request closed without merging**